.github/workflows/run-tests-with-beam.yaml

```diff
@@ -61,7 +61,7 @@ jobs:
           otp: "25"
 
         - os: "ubuntu-22.04"
-          test_erlang_opts: "-s prime_smp"
+          test_erlang_opts: "-s prime_smp,test_gc"
```

Comment:
I'm not about the rationale of this change.

Reply:
Look at the test.  It is unreliable and dependent on the (opaque) behavior of the GC, which is version dependent.  This test started failing on BEAM with these changes.  Probably the best solution is to remove the test_gc test.

Reply:
C.f., PR #1013 (test is failing on`master` as well)